src/pal/tests/palsuite/c_runtime/__iscsym/test1/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ add_executable(paltest_iscsym_test1
 add_dependencies(paltest_iscsym_test1 coreclrpal)
 
 target_link_libraries(paltest_iscsym_test1
-  pthread
```

Comment:
As for the transitivity, I have a feeling that it didn't work for some reason in the past and I had to add it explicitly, but it might be that I am just daydreaming.